src/index.js

```diff
@@ -22,10 +24,14 @@ function App() {
     <>
       <div>
         <Switch>
-          <Route path="/">
+          <Route exact path="/">
```

Comment:
This can also be accomplished by moving this route to the end of the route list. I think this fix is completely fine, but want you to always think about when each plan is more appropriate.